metricsaggregator/src/test/resources/miniwdl-successful-validation-workflow-names.txt

```diff
@@ -0,0 +1,3 @@
+trsId,versionName,isValid,dateExecuted
+#workflow/github.com/garyluu/testWorkflow
```

Comment:
Is this missing 2 column values on purpose (may be in a test, haven't looked).

Reply:
Yeah, it's to verify that the client doesn't send data to Dockstore for lines missing fields